can_tools/utils.py

```diff
@@ -70,11 +70,19 @@ def find_unknown_location_id(df: pd.DataFrame, state_fips: int, engine: Engine =
     return df.loc[~good_rows, :]
 
 
-def find_unknown_demographic_id(df: pd.DataFrame, engine: Engine = None):
+def find_unknown_demographic_id(
+    df: pd.DataFrame, engine: Engine = None, csv_rows=False
+):
     """Find any demographic pairs in the specified dataframe that do not match an entry in the covid_demographics file"""
     if not engine:
         engine = create_dev_engine()[0]
     dems = pd.read_sql("select * from covid_demographics", engine)
     merged = df.merge(dems, on=["sex", "age", "race", "ethnicity"], how="left")
     bad = list(merged["id"].isna())
-    return df.loc[bad, :]
```

Comment:
let's us output the missing demographic pairs as CSV strings like `5-11,all,all,all\n12-19,all,all,all`.

It's still a bit of work as you have to "find and replace" the \n's into actual new lines, but it does make it easier to add the new demographics to the file.

It would be possible (I think) to have this write directly to the `covid_demographics.csv` file, appending any new demographic pairs, but I think having the manual validation is a good thing.

Reply:
Nice!  Yeah, I agree it should be manual.  I was wondering though how hard it would be to make the tests catch the sql constraint blah blah errors and then automatically run something like this and say, "These rows were not found in covid_demographics.csv. Verify they're expected and then add them: ..."  Not urgent or anything though.

Reply:
Yeah I like the sound of that--I'll open a PR after this to add that